src/pages/claim-details/claim-actions.tsx

```diff
@@ -180,7 +180,17 @@ export default function ClaimActions(props: Props) {
               </Tooltip>
             </div>
           </div>
-          <p className={styles.actionMessage}>All done! The claim payout has been accepted.</p>
+          <p className={styles.actionMessage}>
+            All done! The claim payout has been accepted.{' '}
+            <a
```

Comment:
Maybe use the `ExternalLink` compoent?

Reply:
Yeah that's the plan 👍. I've logged a separate issue to make ExternalLink easier to reuse (it's got unnecessary styling that makes it a bit of a pain): https://linear.app/api3-dao/issue/DAO-211/remove-styling-from-externallink-component-to-make-it-easier-to-reuse

As part of that task I will use ExternalLink where appropriate